Cargo.toml

```diff
@@ -219,7 +219,7 @@ strum = { version = "0.26", features = ["derive"] }
 strum_macros = "0.26"
 subtle = "2.6.1"
 test-log = { version = "0.2", features = ["trace"], default-features = false }
-tbs = { package = "fedimint-tbs", path = "./crypto/tbs", version = "=0.7.0-alpha" }
+tbs = { package = "fedimint-tbs", path = "./crypto/tbs", version = "=0.8.0-alpha" }
```

Comment:
Can we just convert all to `workspace = true` now that it will potentially create incompat with the branch anyway?

Reply:
Kinda unrelated to this PR? Although I agree with the direction.

Reply:
Yeah, let's land this and investigate cleaning up the remaining. The last time I attempted it was a can of worms.

https://github.com/fedimint/fedimint/issues/7177